src/defrag-stack.c

```diff
@@ -54,24 +54,15 @@ void DefragTrackerQueueDestroy (DefragTrackerQueue *q)
  *  \param q queue
  *  \param dt tracker
  */
-void DefragTrackerEnqueue (DefragTrackerQueue *q, DefragTracker *dt)
+void DefragTrackerEnqueue(DefragTrackerStack *q, DefragTracker *dt)
```

Comment:
Too late and minor but maybe could be a later cleanup. Comments still say queue and fn names are also `Enqueue` instead of `Push`.